### ISSUE TITLE
Recording more than one window

### ISSUE THREAD

**Issue by Rajagopalan-M** [kapoorlakshya/screen-recorder#83]

**Comment by kapoorlakshya** [comment]
@Rajagopalan-M Can you share more of your code so I can reproduce this locally?

**Comment by Rajagopalan-M** [comment]
@kapoorlakshya 

Hi, it's simple, Just run the automation in 5 browser simultanously and record all the window, this happens. For an example,

> a=[]
> 5.times do
>   a<<Thread.new do
>     require 'watir'
>     @recorder = ScreenRecorder::Window.new(title: ScreenRecorder::Titles.fetch('chrome').last........
>     @recorder.start
>     b.goto "you url"
>     "your code here(make atleast 30 lines of code, so that you can record and see)"
>   end
> end
> 
> a.each(&:join)
>

**Comment by kapoorlakshya** [comment]
@Rajagopalan-M Thanks! 

> the video is not continuously running, it stops at the first page and then suddenly, five page is coming. 

Do you have a different output file for each of the browser sessions?

**Comment by Rajagopalan-M** [comment]
Yes, 5 browsers are being recorded at the same time and it saves 5 files. If you still want an perfect example, I will automate my electricity bill payment flow and give you for your reference. Your video recording software perfectly works for sequential execution but it is parallel execution. If you include this option, then it would be great.

**Comment by kapoorlakshya** [comment]
Yeah, it'll be nice to record parallel test executions.

Can you explain the following behavior a bit more?

> it stops at the first page and then suddenly, five page is coming.

**Comment by Rajagopalan-M** [comment]
Video is moving but you can see only the first page in the video and after such a long time, suddenly last page shows up and then video ends there.

**Comment by kapoorlakshya** [comment]
Got it. I have an idea how to implement support for this, but will need to experiment before I can promise anything.

**Comment by Rajagopalan-M** [comment]
sure thing! Thank you very much for taking this forward
